index.html

```diff
@@ -59,5 +59,218 @@ <h3>LET'S CONNECT</h3>
         </div>
       </div>
     </section>
+
+    <div class="grid-container">
+      <container class="grid-item">
+        <div class="item1">
+          <img src="/images/work1.jpg" alt="Work 1" />
```

Comment:
- [ ] Kindly make sure to fix all image paths. The project currently doesn't display correctly on my device as certain device need the dot at the beginning. Consider updating your image paths from `"/images/work1.jpg"` to `."/images/work1.jpg"`.
**Your project**
![image](https://github.com/peter-mwovi/portfolio_setup/assets/74411135/43c397fa-7af2-4ddb-9421-ec784b2d2a78)

This will make your website much more accessible 😄

Reply:
This issue has been resolved.